src/pmdas/bpf/modules/netatop.c

```diff
@@ -0,0 +1,473 @@
+/*
+ * Based on netatop-bpf:
+ * https://github.com/bytedance/netatop-bpf
+ */
+
+#include "module.h"
+
+#include <assert.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <signal.h>
+#include <string.h>
+#include <errno.h>
+#include <dirent.h>
+#include <sys/resource.h>
+#include <bpf/libbpf.h>
+#include <bpf/bpf.h>
+#include <pcp/pmda.h>
+#include <sys/queue.h>
+#include <pcp/pmwebapi.h>
```

Comment:
should this be <pcp/pmapi.h>?  (not sure - do we use something from webapi here?)
Either way, pmapi.h is included via pmwebapi.h and it includes the headers from line 9 thru 14 I think, so those could be dropped.

Reply:
yep, most of the bpf/modules routines include pmwebapi.h for the pmIniFileLookup dcl so I added it back.  Without it rawhide does not build
netatop.c: In function ‘netatop_init’:
netatop.c:271:16: error: implicit declaration of function ‘pmIniFileLookup’ [-Wimplicit-function-declaration]
  271 |     if ((val = pmIniFileLookup(cfg, module_name, "process_count")))